### PR TITLE
Add prepurchase option docs

### PR DESCRIPTION
**Pull request by necoline**
Adds instructions to set up a team with prepurchased seats
This should only be merged after this [change](https://github.com/codesandbox/codesandbox-client/pull/7455) go out

### CODE REVIEW

File: packages/projects-docs/pages/learn/plan-billing/pricing-faq.mdx [hunk 1]
```diff
@@ -7,16 +7,23 @@ import { Callout } from 'nextra-theme-docs'
 
 # Pricing FAQs
 
+## Can I purchase more than one seat at a time?
+Yes, bulk purchasing of seats is possible on annual subscriptions. 
+If you are creating a team for the first time, select `Custom upgrade` from the button dropdown. 
+![Custom upgrae](../images/subscribe-prepurchase.png)
```
Comment (olarclara) [inline packages/projects-docs/pages/learn/plan-billing/pricing-faq.mdx:13]:
```suggestion
![Custom upgrade](../images/subscribe-prepurchase.png)
```

**Review by filipelima18 — APPROVED** [review]
Made a few wording changes and added a link. Looks good to me.

**Review by filipelima18 — COMMENTED** [review]
Added some assets. Can you check these, @olarclara ?

**Comment by olarclara** [comment]
@filipelima18 Looks great to me! Thank you for updating this :)

**Review by olarclara — APPROVED** [review]

**Pull request merged**